src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx

```diff
@@ -1,51 +1,82 @@
 import { useDropzone } from "react-dropzone";
 import { readFileAsJson } from "../../../common/utils";
+import { vars } from "../../../styles";
 import { ConfigFile } from "../../../types";
+import { ButtonSolidWhiteOrange } from "../../../UI/Button";
+import { Title } from "../../../UI/Title";
+import { Wrapper } from "../../../UI/Wrapper";
 import { ErrorAlert } from "../../Alert";
+import { ProgressBar } from "../../ProgressBar";
+
 interface ConfigFileDropZone {
   errorMessage?: string;
   onConfigFile: (configFile: ConfigFile) => void;
 }
 
-export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = ({ onConfigFile, errorMessage }) => {
-  const [error, setError] = useState(false);
-  const onDrop = async (files: File[]): Promise<void> => {
-    try {
-      const file = files[0];
-      const config = await readFileAsJson<ConfigFile>(file);
-      setError(false);
-      onConfigFile(config);
-    } catch (e) {
-      setError(true);
-      console.error(e);
-    }
-  };
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = styled(
+  ({ className, onConfigFile, errorMessage }) => {
+    const [error, setError] = useState(false);
+    const onDrop = async (files: File[]): Promise<void> => {
+      try {
+        const file = files[0];
+        const config = await readFileAsJson<ConfigFile>(file);
+        setError(false);
+        onConfigFile(config);
+      } catch (e) {
+        setError(true);
+        console.error(e);
+      }
+    };
+    const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
 
-  return (
-    <>
-      <div className="py-3">
-        <h1>Upload Configuration File</h1>
-      </div>
-      {errorMessage && (
-        <div className="my-2">
-          <ErrorAlert message={errorMessage} />
-        </div>
-      )}
-      <div {...getRootProps()}>
-        <input data-testid="config-file-drop-zone" {...getInputProps()} />
-        <div
-          className={`text-center w-100 border-dashed border-2 border-gray-600 p-3 ${
-            isDragActive ? "bg-gray-400" : "bg-white"
-          }`}
-        >
-          {error && <div>Error: File cannot be read</div>}
-          <div>Drag and drop file here</div>
-          <div>or</div>
-          <div>Browse File</div>
-        </div>
+    return (
+      <div className={className}>
+        <Wrapper>
+          <ProgressBar step={1} />
+          <Title>Upload Configuration File</Title>
+          {errorMessage && (
+            <div className="my-2">
+              <ErrorAlert message={errorMessage} />
+            </div>
+          )}
+          <div {...getRootProps()}>
+            <input data-testid="config-file-drop-zone" {...getInputProps()} />
+            <div className={`dropZone ${isDragActive ? "bg-gray-400" : "bg-white"}`}>
+              {error && <div>Error: File cannot be read</div>}
```

Comment:
```suggestion
            <input data-testid="config-file-drop-zone" {...getInputProps()} />
            <div className={`dropZone px-4 text-center ${isDragActive ? "bg-gray-400" : "bg-white"}`}>
              {error && <div>Error: File cannot be read</div>}
```

add abit padding and text-center, this will help look better in small screens

Reply:
ok will implement the change